config/rubocop.yml

```diff
@@ -19,15 +19,14 @@ Style/StringLiterals:
 Metrics/LineLength:
   Max: 100
```

Comment:
Yes - and I will add the additional excludes as a separate PR since they are not related to this PR specifically. I just added the excludes I needed for this PR.